prefect_docker/scripts/flows/workflow_versioning/workflow_versioner.py

```diff
@@ -105,6 +117,21 @@ def create_ingestion_workflow_versioning(transformer=None, mapper=None,
         )
         version_dict[updater_name] = updater
 
+    if refiner:
```

Comment:
Perhaps you can define `service_url` and `endpoint` elsewhere for readability? Maybe it it better to go over the limit in this case.